src/Featurizers/UnitTests/CountVectorizerFeaturizer_UnitTests.cpp

```diff
@@ -72,9 +56,10 @@ TEST_CASE("empty inferencing data - without binary without decorator") {
         ), "'values' is empty"
     );
 }
+
 TEST_CASE("string - without binary, without decorator, analyze word, maxdf = 1, mindf = 0, topk = null, empty vocabulary, ngram_min = 1, ngram_max = 1") {
     using InputType       = std::string;
-    using TransformedType = NS::Featurizers::SparseVectorEncoding<std::uint32_t>;
+    using TransformedType = NS::Featurizers::SparseVectorEncoding<std::float_t>;
```

Comment:
>float_t [](start = 71, length = 7)

Will CountVectorizer ever return a value that is not a whole number? I think that CountVectorizer should retain the existing signature; values should be converted from what is returned by the underlying implementation.